src/evm/WormholeVerifier.sol

```diff
@@ -1429,15 +1462,18 @@ contract WormholeVerifier is EIP712Encoding {
   }
 
   function _storeSchnorrShardDataBlock(uint32 schnorrKeyIndex, bytes memory shardData) internal {
-    uint256 shardCount = shardData.length >> 6;
+    uint256 shardCount = shardData.length / 96;
```

Comment:
I get the feeling that this could be further improved but I'm unsure how right now.
```suggestion
    uint256 shardCount = shardData.length / (3 * LENGTH_WORD);
```